frontend/src/scenes/trends/Trends.tsx

```diff
@@ -27,9 +27,9 @@ export function TrendInsight({ view }: Props): JSX.Element {
     const { cohortModalVisible } = useValues(personsModalLogic)
     const { setCohortModalVisible } = useActions(personsModalLogic)
     const { filters: _filters, loadMoreBreakdownUrl, breakdownValuesLoading } = useValues(
-        trendsLogic({ dashboardItemId: null, view, filters: null })
+        trendsLogic({ dashboardItemId: null, filters: null })
     )
-    const { loadMoreBreakdownValues } = useActions(trendsLogic({ dashboardItemId: null, view, filters: null }))
```

Comment:
This wasn't used at all

Reply:
Quick q for curiosity, why do we reset the logic props here?